device/transport/mqtt/devdoc/mqtt_requirements.md

```diff
@@ -266,3 +266,34 @@ The `getTwinReceiver` method creates a `MqttTwinReceiver` object for the twin re
 **SRS_NODE_DEVICE_MQTT_18_007: [** If a twin receiver for this endpoint previously existed, the `getTwinReceiver` method should return the preexisting `MqttTwinReceiver` object as the second parameter of the `done` function with null as the first parameter. **]**
 
 
+### enableC2D
+
+**SRS_NODE_DEVICE_MQTT_16_047: [** `enableC2D` shall connect the MQTT connection if it is disconnected. **]**
+
+**SRS_NODE_DEVICE_MQTT_16_048: [** `enableC2D` shall calls its callback with an `Error` object if it fails to connect. **]**
+
+**SRS_NODE_DEVICE_MQTT_16_049: [** `enableC2D` shall subscribe to the MQTT topic for messages. **]**
```

Comment:
>`enableC2D` shall subscribe to the MQTT topic for messages [](start = 35, length = 58)

What if the subscription fails or times out?  I'm not sure how likely this is, but both could theoretically happen.  Same enableMethods.  If there's an UNSUBACK, then this also applies to disableC2D and disableMethods.